showcase/app/components/mock/app/sidebar/side-nav.gts

```diff
@@ -19,7 +19,7 @@ import {
 import type { HdsSideNavSignature } from '@hashicorp/design-system-components/components/hds/side-nav/index';
 import type Owner from '@ember/owner';
 
-export interface MockAppSidebarSideNavSignature {
+export interface MockSidebarSideNavSignature {
```

Comment:
[though] what if, to avoid conflicts, we keep the `App` after `Mock`, but we add `Old`? Something like: `MockAppSidebarOldSideNav`